pylsp/plugins/hover.py

```diff
@@ -41,10 +41,19 @@ def pylsp_hover(config, document, position):
         "",
     )
 
+    include_docstring = signature_config.get("include_docstring", True)
+
+    # raw docstring returns only doc, without signature
+    docstring = definition.docstring(raw=True)
+    if include_docstring is False:
```

Comment:
```suggestion
    if not include_docstring:
```

This is a bit more Pythonic and easier to read.

Reply:
Done